Automattic/jetpack
# My Jetpack: Handling connection notices

Closes #22648 

#### Changes proposed in this Pull Request:

* Expose `hasConnectedOwner` from `useConnection`.
* Create a new hook called `useConnectionWatcher` that watches for site and user connection.
* Remove `useNoticeWatcher` hook.
* Remove site connection handling for `useMyJetpackConnection`.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Build `My Jetpack` (`jetpack build packages/my-jetpack`)
* Disconnect only user.
* Make sure that you have only 1 product that requires user connection (Backup, Scan, Search) activated.
* You should see the message `Jetpack ${product} ...`
* Now activate 1 more product that requires user connection.
* You should see the message `Some products ...`

#### DEMO


https://user-images.githubusercontent.com/1663717/153016937-43c6333c-d18e-443c-90df-b9e4d50dc6c4.mp4
